articles/postgresql/concepts-extensions.md

```diff
@@ -14,24 +14,24 @@ ms.date: 09/26/2017
 PostgreSQL provides the ability to extend the functionality of your database using extensions. Extensions allow for bundling multiple related SQL objects together in a single package that can be loaded or removed from your database with a single command. After being loaded in the database, extensions can function as do built-in features. For more information on PostgreSQL extensions, see [Packaging Related Objects into an Extension](https://www.postgresql.org/docs/9.6/static/extend-extensions.html).
 
 ## How to use PostgreSQL extensions
-PostgreSQL extensions must be installed for your database before you can use them. To install a particular extension, run the [CREATE EXTENSION](https://www.postgresql.org/docs/9.6/static/sql-createextension.html) command from psql tool to load the packaged objects into your database.
+PostgreSQL extensions must be installed in your database before you can use them. To install a particular extension, run the [CREATE EXTENSION](https://www.postgresql.org/docs/9.6/static/sql-createextension.html) command from psql tool to load the packaged objects into your database.
 
-Azure Database for PostgreSQL supports a subset of key extensions as listed here. Extensions beyond the ones listed are not supported; you cannot create your own extension with Azure Database for PostgreSQL service.
+Azure Database for PostgreSQL supports a subset of key extensions as listed below. Extensions beyond the ones listed are not yet supported; you cannot create your own extension with Azure Database for PostgreSQL service.
```

Comment:
The next paragraph states "that are currently supported", so this implies that extensions not listed, are not YET supported.

Reply:
Microsoft may never support some extensions, so _yet_ implies optimism that may not be true.

Reply:
The list of supported extension has increased in the past two weeks, so wanted to make sure it didn't imply that this is all that will ever be supported.

How about removing the "yet", but change the previous sentence to add "currently":

Azure Database for PostgreSQL currently supports a subset of key extensions as listed below. Extensions beyond the ones listed are not supported; you cannot create your own extension with Azure Database for PostgreSQL service.

Reply:
"Currently" works better for me.  The Next Steps had a way to give feedback to that point.